test/Core.Test/Tools/Services/NonAnonymousSendCommandTests.cs

```diff
@@ -0,0 +1,1157 @@
+            .Throws(new BadRequestException("Due to an Enterprise Policy, you are only able to delete an existing Send."));
+
+        // Act & Assert
+        var exception = await Assert.ThrowsAsync<BadRequestException>(() =>
+            _nonAnonymousSendCommand.SaveSendAsync(send));
+
+        Assert.Contains("Enterprise Policy", exception.Message);
+
+        // Verify the validation service was called
+        await _sendValidationService.Received(1).ValidateUserCanSaveAsync(send.UserId.Value, send);
+
+        // Verify repository was not called since exception was thrown
+        await _sendRepository.DidNotReceive().CreateAsync(Arg.Any<Send>());
+    }
+
+    [Theory]
+    [InlineData(true)]  // New Send (Id is default)
+    [InlineData(false)] // Existing Send (Id is not default)
+    public async Task SaveSendAsync_DisableSend_DoesntApply_success(bool isNewSend)
+    {
+        // Arrange
+        var userId = Guid.NewGuid();
+        var send = new Send
+        {
+            Id = isNewSend ? default : Guid.NewGuid(),
+            Type = SendType.Text,
+            UserId = userId,
+            Data = "Text with Notes"
+        };
+
+        var initialDate = DateTime.UtcNow.AddMinutes(-5);
+        send.RevisionDate = initialDate;
+
+        // Configure validation service to NOT throw (policy doesn't apply)
+        _sendValidationService.ValidateUserCanSaveAsync(userId, send).Returns(Task.CompletedTask);
+
+        // Set up context for reference event
+        _currentContext.ClientId.Returns("test-client");
+        _currentContext.ClientVersion.Returns(Version.Parse("1.0.0"));
+
+        // Act
+        await _nonAnonymousSendCommand.SaveSendAsync(send);
+
+        // Assert
+        // Verify validation was checked
+        await _sendValidationService.Received(1).ValidateUserCanSaveAsync(userId, send);
+
+        if (isNewSend)
+        {
+            // For new Sends
+            await _sendRepository.Received(1).CreateAsync(send);
+            await _pushNotificationService.Received(1).PushSyncSendCreateAsync(send);
+            await _referenceEventService.Received(1).RaiseEventAsync(Arg.Is<ReferenceEvent>(e =>
+                e.Id == userId &&
+                e.Type == ReferenceEventType.SendCreated &&
+                e.Source == ReferenceEventSource.User &&
+                e.SendType == send.Type &&
+                e.SendHasNotes == true &&
+                e.ClientId == "test-client" &&
+                e.ClientVersion == Version.Parse("1.0.0")));
+        }
+        else
+        {
+            // For existing Sends
+            await _sendRepository.Received(1).UpsertAsync(send);
+            Assert.NotEqual(initialDate, send.RevisionDate);
+            await _pushNotificationService.Received(1).PushSyncSendUpdateAsync(send);
+            await _referenceEventService.DidNotReceive().RaiseEventAsync(Arg.Any<ReferenceEvent>());
+        }
+    }
+
+    // TODO: REMOVE
```

Comment:
⛏️ Is this just a comment that should've been removed?

Reply:
Fixed.